os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -1056,7 +1127,7 @@ void i2s_transfer_rx_handleirq(void *data, char *pbuf)
 
 	int result = OK;
 
-	i2s_rxdma_callback(priv, result);
+	//i2s_rxdma_callback(priv, result);
```

Comment:
Do you need commented codes? I find several commented codes.

Reply:
Ok, I have removed the commented codes that are not needed, such as i2s_txdma_prep()